src/commands/BootstrapCommand.js

```diff
@@ -107,7 +107,9 @@ export default class BootstrapCommand extends Command {
         // postinstall bootstrapped packages
         (cb) => this.postinstallPackages(cb),
         // prepublish bootstrapped packages
-        (cb) => this.prepublishPackages(cb)
+        (cb) => this.prepublishPackages(cb),
+        // prepare bootstrapped packages
+        (cb) => this.preparePackages(cb)
```

Comment:
dont think `bootstrap` needs any more work besides the tests, please correct me if im wrong